app/controllers/admin/questions_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Admin::QuestionsController < AdminController
+  def index
+    @bank_questions = Question.for_banks
+    @partner_questions = Question.for_partners
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def create
+    @question = Question.create(question_params)
+    if @question.valid?
+      redirect_to admin_questions_path
+    else
+      flash[:error] = "Failed to create question. #{error_message_for_answer(@question.errors)}"
+      render :new
+    end
+  end
+
+  def edit
+    @question = current_question
+  end
+
+  def update
+    @question = current_question
+    @question.update(question_params)
+    if @question.valid?
```

Comment:
```suggestion
    if @question.update(question_params)
```
`update` will also trigger a `valid?` call and populate the errors.

Reply:
I refactored it the way that you recommended.